todo_app/mongo/mongo_wrapper.py

```diff
@@ -4,19 +4,21 @@
 from bson.objectid import ObjectId
 import os
 
+TODO_BASE = "_todo"
+COMPLETED_BASE = "_completed"
+DOING_BASE = "_doing"
 
 class MongoWrapper:
 
-    def __init__(self, database: str = ""):
-        
-        client = self._connect(database)
+    def __init__(self, connection_string, database):
+        client = self._connect(f"{connection_string}?ssl=true&retrywrites=false&replicaSet=globaldb&maxIdleTimeMS=120000&{os.environ.get('APP_NAME')}")
         self.db = client.database
+        self.todo = self.db[f"{database}{TODO_BASE}"]
+        self.completed = self.db[f"{database}{COMPLETED_BASE}"]
+        self.doing = self.db[f"{database}{DOING_BASE}"]

```

Comment:
I think this is accessing a database called "database" (the string), then reading from collections called `"{database}{TODO_BASE}"` (the variable database).

Is that intentional? If so it's probably worth renaming the `database` variable.

Reply:
Yes, this is the case. Due to the way I handle the connection string use different collection names for my "test" database and "prod" database rather than creating actually new databases.